spec.bs

```diff
@@ -3474,6 +3474,10 @@ Each {{InterestGroupReportingScriptRunnerGlobalScope}} has a
 <div algorithm>
   The <dfn method for="InterestGroupReportingScriptRunnerGlobalScope">registerAdMacro(|name|,
   |value|)</dfn> method steps are:
+  1. [=list/For each=] [=string/code unit=] |c| in |name| and |value|:

```

Comment:
I think it's just [=code unit=].  (oh you already made the change 1 min ahead of me)